desktop/Automattic/wp-desktop/bin/build-desktop-secrets.js

```diff
@@ -40,6 +41,9 @@ for ( let i = 0; i < secrets.length; i++ ) {
 	}
 
 	try {
+		if ( ! fs.existsSync( path.resolve( PROJECT_DIR, 'config' ) ) ) {
```

Comment:
Can be replaced with `fs.mkdirSync( path.resolve( PROJECT_DIR, 'config' ) , { recursive: true } )`, it will take care of checking if the dir exists before trying to create it.